tools/docker/runner/entrypoint.sh

```diff
@@ -11,13 +11,24 @@ run() {
     "$@" || exit $?
```

Comment:
The commit message says "we could use any of the techniques
in [1]" but there is no "[1]".
Did you forget to add it?